src/pages/LogIn/components/LoginForm.tsx

```diff
@@ -104,6 +104,7 @@ const LoginFormComponent: React.FC<AllProps> = ({ setAuthToken }: AllProps) => {
       isValidUsername={isValidUsername}
       passwordLabel="Password"
       passwordValue={passwordValue}
+      isShowPasswordEnabled
```

Comment:
Well done @aparna-bhatt   cc @mairin @jennydaman

Reply:
Thank you @sherrif10 .